src/sonic-config-engine/portconfig.py

```diff
@@ -259,6 +260,13 @@ def __init__(self, name, bmode, properties):
         if not self._breakout_capabilities:
             raise RuntimeError("Unsupported breakout mode {}!".format(bmode))
 
+    def _parse_name_to_prefix_and_id(self):
+        match = re.match(swsscommon.FRONT_PANEL_PORT_REGEX, self._name)
+        try:
+            return match.group(1), int(match.group(2))
+        except Exception:
```

Comment:
>Exception [](http://example.com/codeflow?start=15&length=9)

Please use more specific exception type.

Reply:
I removed the exception due to the above comment